apps/angular/src/app/features/anime-table/anime-table.component.html

```diff
@@ -0,0 +1,23 @@
+<div>
+  <table class="table">
+    <caption class="table__caption">Anime table</caption>
+    <tr class="table__row table__row_caption">
+      <th scope="row" class="table__row_item">Thumbnail</th>
+      <th scope="row" class="table__row_item">English title</th>
+      <th scope="row" class="table__row_item">Japanese title</th>
+      <th scope="row" class="table__row_item">Aired start</th>
+      <th scope="row" class="table__row_item">Type</th>
+      <th scope="row" class="table__row_item">Status</th>
+    </tr>
+    <tr *ngFor="let item of (result$ | async)?.results" class="table__row">
+      <th scope="row" class="table__row_item table__row_item_thumb">
+        <img class= "table__row_item_thumb__img" src="{{item.image}}" alt="{{item.titleEnglish}}" />
```

Comment:
Let's use angular [binding syntax](https://angular.io/guide/binding-syntax) over mustache syntax
```suggestion
        <img class= "table__row_item_thumb__img" [src]="item.image" [alt]="item.titleEnglish" />
```